convert/lib/convert.py

```diff
@@ -0,0 +1,614 @@
+#!/usr/bin/env python3
+import argparse
+import boto3
+import json
+import re
+import yaml
+from copy import deepcopy
+from functools import reduce
+from pprint import pprint
+from subprocess import run
+from time import sleep
+
+from .meta import (
+    cdk_ids,
+    cf_status,
+    resource_template,
+    efs_backup_resources,
+    route53_resource,
+    stack_map,
+)
+from .nuke import nuke
+
+
+clean_categories = [x.name for x in cdk_ids if x.name != "cloudformation_stack"]
+
+
+class app:
+    def __init__(self):
+        self.parse_args()
+        self.args.command()
+
+    def get_stacks(self, stack: str = None, full: bool = False):
+        stacks = {"resources": {}}
+        for s in self.cf.describe_stack_resources(StackName=stack or self.stack_name)[
+            "StackResources"
+        ]:
+            logical_id = s["LogicalResourceId"]
+            physical_id = s["PhysicalResourceId"]
+            if s["ResourceType"] == cdk_ids.cloudformation_stack.value:
+                for mapped_logical_id, name in stack_map.items():
+                    if logical_id.startswith(mapped_logical_id):
+                        stacks[name] = self.get_stacks(physical_id, full)
+                        break
+                else:
+                    raise Exception(f"Nothing to map stack {s} to!")
+            else:
+                if re.match("^[0-9A-F]{8}", logical_id[-8:]):
+                    stacks["resources"][logical_id[:-8]] = s if full else physical_id
+                else:
+                    stacks["resources"][logical_id] = s if full else physical_id
```

Comment:
just a bit cleaner IMO

```suggestion
                key = local-id[:-9] if re.match("^[0-9A-F]{8}", logical_id[-8:]) else logical_id
                stacks["resources"][key] = s if full else physical_id
```